internal/operator-controller/features/features.go

```diff
@@ -11,13 +11,11 @@ import (
 const (
 	// Add new feature gates constants (strings)
 	// Ex: SomeFeature featuregate.Feature = "SomeFeature"
-	PreflightPermissions              featuregate.Feature = "PreflightPermissions"
-	SingleOwnNamespaceInstallSupport  featuregate.Feature = "SingleOwnNamespaceInstallSupport"
-	SyntheticPermissions              featuregate.Feature = "SyntheticPermissions"
-	WebhookProviderCertManager        featuregate.Feature = "WebhookProviderCertManager"
-	WebhookProviderOpenshiftServiceCA featuregate.Feature = "WebhookProviderOpenshiftServiceCA"
```

Comment:
Same here, we can't really delete feature-gates until it's no longer referenced downstream.